lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -21,25 +29,84 @@ type Props = {
   addStopToPattern: typeof stopStrategiesActions.addStopToPattern,
   // property is available through react dnd?
   connectDragSource: any,
+  controlLabel: string,
```

Comment:
I would separate the props for the new component `<PickupDropoffSelect>` (see #691).
`Props` also still combine props from two components, but separating those is not relevant to this PR, so I am not recommending to separate them (other might).